lib/lexisnexis/threat_metrix/proofer.rb

```diff
@@ -0,0 +1,30 @@
+module LexisNexis
+  # Verifies through the DDP platform (newer)
+  class ThreatMetrix
+    class Proofer < LexisNexis::Proofer
+      vendor_name 'lexisnexis:threat_metrix_ddp'
```

Comment:
So I think this is the best name..... but open to feedback

Some thoughts:
- I *hate* the "e" in Metrix --- it's 100% their actual product name, but I know this is going to get typo'd somewhere and I'm worried it'll be typod
- I wanted to include `ddp` because the other platform is `rdp` and I can never which is which so I figured this was the clearest we could be?